tests/src/integration/options.cpp

```diff
@@ -48,7 +48,7 @@ std::string Options::public_key_ = "public.key";
 std::string Options::private_key_ = "private.key";
 bool Options::is_verbose_ccm_ = false;
 bool Options::is_verbose_integration_ = false;
-bool Options::is_beta_protocol_ = true;
+bool Options::is_beta_protocol_ = false;
```

Comment:
Not strictly necessary (the fix above is more than adequate to address the problem) but more correct given the current state of things.  The driver will not recognize v5-beta in other integration tests but since v4 is the highest protocol version we officially support this _shouldn't_ be a major obstacle.